src/pages/projects.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import MainLayout from "../layouts/MainLayout"
+import SEO from "../components/SEO"
+import Header from '../components/Header'
+import ProjectsList from '../components/ProjectsList'
+import {useProjectsPageData} from '../queries/projects'
+
+const ProjectsPage = ({ data }) => {
+  const edges = useProjectsPageData()
+  const projects = edges.map(edge => ({ id: edge.node.id, ...edge.node.frontmatter}))
+  console.log(projects)
```

Comment:
Let's remove `console.log`s. Do we have a linter in place?

Reply:
Sure will do that! Linter is set up but needs to alter the rules! I'll do that too